Packs/Ataya/Integrations/Ataya/Ataya.py

```diff
@@ -0,0 +1,111 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import urllib3
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    def __init__(self, api_key: str, base_url: str, proxy: bool, verify: bool):
+        super().__init__(base_url=base_url, proxy=proxy, verify=verify)
+        self.api_key = api_key
+
+        self._headers = {
+            'Content-Type': 'application/json',
+            'x-api-key': self.api_key
+        }
+
+    def getNode(self):
+        return self._http_request(method='GET', url_suffix='api/v1/mgmt/5gc/networks/default/nodes',
+                                  resp_type='json', ok_codes=(200,))
+
+    def assignUser(self, imsi):
+        return self._http_request(method='PUT', url_suffix='api/v1/mgmt/5gc/clientAction/setstatus',
+                                  json_data={"status": "assigned", "resources": [imsi]}, resp_type='json', ok_codes=(200,))
+
+
+''' HELPER FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """
+    Tests API connectivity and authentication'
+    Returning 'ok' indicates that connection to the service is successful.
+    Raises exceptions if something goes wrong.
+    """
+
+    try:
+        response = client.getNode()
+
+        success = demisto.get(response, 'count')  # Safe access to response['success']['total']
+        if success < 1:
+            return f'Unexpected result from the service: success={success} (expected success > 1)'
+
+        return 'ok'
+
+    except Exception as e:
+        exception_text = str(e).lower()
+        if 'forbidden' in exception_text or 'authorization' in exception_text:
+            return 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def assign_command(client: Client, imsi: str):
+    if not imsi:
+        raise DemistoException('the imsi argument cannot be empty.')
+
+    response = client.assignUser(imsi=imsi)
+    userStatus = demisto.get(response, 'status')
+
+    if userStatus == 'unassigned':
+        raise DemistoException('Assign User Fail', res=response)
+
+    return f'User {imsi} {userStatus}'
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
```

Comment:
No need to run coverage on the `main` function:

```suggestion
def main() -> None:  # pragma: no cover
```